src/components/plots/CategoricalEmbeddingPlot.jsx

```diff
@@ -38,7 +38,10 @@ const CategoricalEmbeddingPlot = (props) => {
       dispatch(loadEmbedding(experimentId, embeddingSettings.method));
     }
   }, [experimentId, embeddingSettings.method]);
-
+  useEffect(() => {
+    const spec = generateSpec(config);
+    setPlotSpec(spec);
+  }, [config]);
```

Comment:
why was this needed?

Reply:
when we reset the plot, the spec was not changed with the new config.